src/services/milestones/milestones.hooks.js

```diff
@@ -61,6 +62,107 @@ const restrict = () => (context) => {
     .then(milestones => ((Array.isArray(milestones)) ? milestones.forEach(canUpdate) : canUpdate(milestones)));
 };
 
+
+/**
+ *
+ * Conditionally sends a notification after patch or create
+ *
+ **/
+const sendNotification = () => (context) => {
+  const { id, data, app, result } = context;
+
+  /**
+   * Notifications when a milestone get created
+   **/
+  if(context.method === 'create') {
+    if(result.status === 'proposed') {
+      // find the campaign admin and send a notification that milestone is proposed
+      app.service('users').find({query: { address: data.campaignOwnerAddress }})
+        .then((users) => {
+          console.log('campaignOwner', users)
+
+          Notifications.milestoneProposed(app, {
+            recipient: users.data[0].email,
+            user: users.data[0].name,
+            milestoneTitle: data.title,
+            campaignTitle: result.campaign.title,
+            amount: data.maxAmount
+          });  
+        })
+        .catch((e) => console.error('error sending proposed milestone notification', e));
```

Comment:
lets use:

```
import logger from 'winston';
logger.info/debug/etc
```
instead of `console.log`. we need to replace all instances of `console.log` in this PR